lib/appsignal/integrations/dry_monitor.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+module Appsignal
+  module Integrations
+    module DryMonitorIntegration
+      def instrument(event_id, payload = {}, &block)
+        Appsignal::Transaction.current.start_event
+
+        super
+      ensure
+        title, body, body_format = Appsignal::EventFormatter.format("#{event_id}.dry", payload)
+
+        Appsignal::Transaction.current.finish_event(
+          title || event_id.to_s,
+          title,
```

Comment:
In the formatted event test, both `title` and `name` are set to `"query.postgres"`, but in the test for the generic event one of them is left empty. Is that the intended behaviour? If not:

```suggestion
          title || event_id.to_s,
```

Reply:
We might also want to provide a `.dry` suffix for the resulting event category -- although we don't currently do this for ActiveSupport notifications.

Reply:
I wouldn't add dry to the event category. It's the thing that reports it, but no one cares about that detail.